src/feedback-rating/feedback-rating-data.ts

```diff
@@ -0,0 +1,6 @@
+export const FeedbackRatingData = {
+    IMG: "https://assets.life.gov.sg/react-design-system/img/feedback-rating/feedback-rating-banner.png",
+    IMG_ALT: "feedback-rating-banner",
```

Comment:
can set alt to an empty string as the image is purely decorative

Reply:
Updated. Removed alt, and removed broken icon onError